.stylelintrc.json

```diff
@@ -30,7 +30,7 @@
     "alpha-value-notation": ["percentage"],
     "hue-degree-notation": ["number"],
     "color-function-notation": ["modern"],
-    "length-zero-no-unit": [true],
+    "length-zero-no-unit": [true, "custom-properties"],
```

Comment:
I'm not sure if this is a valid configuration in your version of `stylelint` already, but just today I ran into the issue that they've changed the configuration API for `custom-properties` in their latest release:

https://github.com/nl-design-system/rijkshuisstijl-design-tokens/blob/d6add187b5bf3f8398374e85e07322d75e7c7ae5/.stylelintrc.json#L33

If it is compatible with the old version, you could already use the new syntax instead.

Reply:
That explains a lot! We've been messing around with that stylelint rule since it was causing an error on some branches. I'm guessing this is the cause.

Reply:
Yeah makes sense! So either you can encourage all developers to install locally with `yarn --frozen-lockfile` which is the yarn variant of `npm ci`.

The other thing you could do is to avoid depending on unreliable version ranges, because it requires the developers of every dependency to follow semantic versioning perfectly, which is unrealistic.

In `npm` projects I usually configure `save-exact=false` in the `.npmrc` file, and I think the `.yarnrc` equivalent is `save-prefix ""`.

In any case I would suggest to use `yarn --frozen-lockfile` instead of just `yarn` in your GitHub workflows:
https://github.com/Gemeente-DenHaag/denhaag-component-library/blob/d6dcf57560e3c98f18a3cba64ceb2936803ef5fd/.github/workflows/cicd.yml#L28

This should help achieving reproducible builds.

Reply:
I'd like @mjcarsjens input on this. I recall the current npm config being a pretty well discussed choice. I don't (yet :D) have the experience to tell whether or not setting save-prefix is beneficial for us without looking into it more deeply. I did find a github issue that weighs the benefits for making it a defualt (https://github.com/yarnpkg/yarn/issues/4147) 

In any case, the linter error has been fixed, thanks!